src/Notes/WC_Admin_Notes_Marketing.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * WooCommerce Admin Marketing Note Provider.
+ *
+ * Adds notes to the merchant's inbox concerning the marketing dashboard.
+ *
+ * @package WooCommerce Admin
+ */
+
+namespace Automattic\WooCommerce\Admin\Notes;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * WC_Admin_Notes_Marketing
+ */
+class WC_Admin_Notes_Marketing {
+
+	/**
+	 * Note traits.
+	 */
+	use NoteTraits;
+
+	/**
+	 * Name of the note for use in the database.
+	 */
+	const NOTE_NAME_INTRO = 'wc-admin-marketing-intro';
+
+	/**
+	 * Attach hooks.
+	 */
+	public function __construct() {
+		add_action( 'woocommerce_note_action_install-now', array( $this, 'install' ) );
```

Comment:
Looks like there isn't an `install()` method in this class.

Reply:
Thanks @jeffstieler good catch, copy and past fail.Fixed with 680c05c